setup_local_environnement.md

```diff
@@ -0,0 +1,24 @@
+# Setup your local environnement
```

Comment:
environ~ne~ment

Reply:
Ignore this comment. I'd like to include just the bootstrap_form update in this PR.

Reply:
Sorry it's a french word... Do you prefer to include on this file on [this issue](https://github.com/tomatoes-app/tomatoes/issues/218) or simply remove it?

Reply:
Before working on that issue, I would like to complete https://github.com/tomatoes-app/tomatoes/issues/128 to support Docker. Just remove this file for now please.

Reply:
I removed file https://github.com/tomatoes-app/tomatoes/pull/299/commits/41c51892ee1ffa2753cc856b4afc075cf8d1e55a